gistUploader.js

```diff
@@ -1,11 +1,24 @@
 
 const axios = require('axios');
+const fs = require('fs');
 
-axios.post(
-  'https://api.github.com/gists',
-  {
-    'files': { 'short.txt': { 'content': 'file contents' } },
-  },
-)
-  .then(response => (console.log(response)))
-  .catch(error => (console.log(error)));
+let data;
+
+function processFile(fileToPost) {
+  axios.post(
+    'https://api.github.com/gists',
+    {
+      files: { 'short.txt': { content: fileToPost } },
+    },
+  )
+    .then(response => (console.log(response)))
+    .catch(error => (console.log(error)));
+}
+
+fs.readFile(process.argv[2], (error, contents) => {
+  if (error) {
+    console.log(error);
+  }
+  data = contents.toString();
```

Comment:
no need for the assignment here, just pass this into the func

Reply:
done